repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/identityv3/cache/InvalidateV3CacheUsingAuthenticatedFeedTest.groovy

```diff
@@ -0,0 +1,217 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package features.filters.identityv3.cache
+
+import features.filters.keystonev2.AtomFeedResponseSimulator
+import framework.ReposeValveTest
+import framework.category.Slow
+import framework.mocks.MockIdentityV3Service
+import org.junit.experimental.categories.Category
+import org.rackspace.deproxy.Deproxy
+import org.rackspace.deproxy.Endpoint
+import org.rackspace.deproxy.MessageChain
+import org.rackspace.deproxy.Response
+
+@Category(Slow.class)
+class InvalidateV3CacheUsingAuthenticatedFeedTest extends ReposeValveTest {
+    Endpoint originEndpoint
+    Endpoint atomEndpoint
+    Endpoint identityEndpoint
+    AtomFeedResponseSimulator fakeAtomFeed
+    MockIdentityV3Service fakeIdentityV3Service
+
+    def setup() {
+        deproxy = new Deproxy()
+
+        int atomPort = properties.atomPort
+        fakeAtomFeed = new AtomFeedResponseSimulator(atomPort)
+        atomEndpoint = deproxy.addEndpoint(atomPort, 'atom service', null, fakeAtomFeed.handler)
+
+        def params = properties.defaultTemplateParams
+        repose.configurationProvider.applyConfigs("common", params)
+        repose.configurationProvider.applyConfigs("features/filters/identityv3/common", params)
+        repose.configurationProvider.applyConfigs("features/filters/identityv3/atom", params)
+        repose.configurationProvider.applyConfigs("features/filters/identityv3/atom/wauthenticatedfeed", params)
+        repose.start()
+
+        originEndpoint = deproxy.addEndpoint(properties.targetPort, 'origin service')
+        fakeIdentityV3Service = new MockIdentityV3Service(properties.identityPort, properties.targetPort)
```

Comment:
@Mario-Lopez I think this might be why you're seeing the issue with authenticated feeds. The Atom Feed Service only supports Keystone v2 right now, but the mock service you're using here is Identity v3. Particularly confusing, since you're testing the Keystone v3 filter. ;)